react-components/src/components/i18n/I18n.tsx

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+import { useEffect, createContext, useContext, useState, type ReactElement } from 'react';
+
+import { type I18nProps, type I18nContent, type Translations } from './types';
+import { getLanguage } from './utils';
+
+import en from '../../common/i18n/en/reveal-react-components.json';
```

Comment:
Is there a way for us to do this automatically? Right now, we need to manually add new languages when we receive support for them.